corehq/apps/export/forms.py

```diff
@@ -160,24 +160,24 @@ def __init__(self, has_form_export_permissions, has_case_export_permissions, *ar
                 ng_show="!staticModelType"
             ),
             crispy.Div(
-                crispy.Div(
+                crispy.Div(  # Form export fields
+                    crispy.Div(
+                        crispy.Field(
+                            'app_type',
+                            placeholder=_("Select Application Type"),
+                            ng_model="formData.app_type",
+                            ng_change="updateAppChoices()",
+                            ng_required="true",
+                        ),
+                        ng_show="hasSpecialAppTypes && formData.model_type === 'form'",
+                    ),
                     crispy.Field(
-                        'app_type',
-                        placeholder=_("Select Application Type"),
-                        ng_model="formData.app_type",
-                        ng_change="updateAppChoices()",
-                        ng_required="true",
+                        'application',
+                        placeholder=_("Select Application"),
+                        ng_model="formData.application",
+                        ng_change="formData.model_type === updateModuleChoices()",
```

Comment:
Comparing this with old line 177, this looks like it should just be `updateModuleChoices()`...?

Reply:
you're right - will update.